README.md

```diff
@@ -43,7 +43,9 @@ We strive for collaboration with [mutual respect for each other][contributing].
 
 ### Discussion
```

Comment:
Perhaps we also add the badge to the top:

![screen shot 2017-01-24 at 8 21 52 pm](https://cloud.githubusercontent.com/assets/254562/22274162/cde22f74-e272-11e6-9392-a06d8b09d269.png)